l10n.yaml

```diff
@@ -0,0 +1,6 @@
+arb-dir: lib/l10n
```

Comment:
```suggestion
# Docs on this config file:
#   https://docs.flutter.dev/ui/accessibility-and-localization/internationalization#configuring-the-l10nyaml-file

arb-dir: lib/l10n
```

That'll make it a bit smoother whenever we're adjusting these settings. (I came across that documentation because it's on the same page as the tutorial linked from #275, and I was rereading that.)